client/config_parser.py

```diff
@@ -16,7 +16,11 @@
 import os
 
 from argparse import Namespace
+<<<<<<< HEAD
+from os.path import exists, abspath
+=======
 from object_mapping_parser import ObjectMappingParser
+>>>>>>> main
```

Comment:
Unresolved conflict.

Reply:
I missed that. Thanks for catching it. Done.